examples/cifar10/configs/cifar10_scaffold/config/config_fed_server.json

```diff
@@ -0,0 +1,87 @@
+{
+  "format_version": 2,
+
+  "min_clients": 8,
+  "num_rounds": 50,
+
+  "server": {
+    "heart_beat_timeout": 600
+  },
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
+      "name": "PTFileModelPersistor",
+      "args": {
+        "model": {
+          "path": "pt.networks.cifar10_nets.ModerateCNN",
+	      "args": {}
+        }
+      }
+    },
+    {
+      "id": "shareable_generator",
+      "name": "FullModelShareableGenerator",
+      "args": {}
+    },
+    {
+      "id": "aggregator",
+      "name": "InTimeAccumulateWeightedAggregator",
+      "args": {
+        "expected_data_kind": {
+          "_model_weights_": "WEIGHT_DIFF",
+          "scaffold_c_diff": "WEIGHT_DIFF"
+        }
+      }
+    },
+    {
+      "id": "model_selector",
+      "name": "IntimeModelSelectionHandler",
+      "args": {}
+    },
+    {
+      "id": "model_locator",
+      "path": "pt.pt_model_locator.PTModelLocator",
```

Comment:
You can now use the built-in PTFileModelLocator in nvflare/app_common/pt/pt_file_model_locator.py.
You can remove the one in your custom folder.

Reply:
removed the custom version.